### ISSUE TITLE
Tridactyl doesn't work on theguardian.com without uBlock

### ISSUE THREAD

**Issue by rddunphy** [tridactyl/tridactyl#4312]
Tridactyl version: 1.22.1pre6337-190f003e
Firefox version: Mozilla Firefox 102.0
Operating system: linux

-   Brief description of the problem: Tridactyl isn't loaded on the [Guardian website](https://www.theguardian.com/uk) and some but not all subpages (article pages seem to be fine). There don't appear to be any relevant errors in the console, and I can't see the iframe or mode indicator span in the inspector. The escape hatch doesn't work, either (with `escapehatchsidebarhack` set to `true`).

-   URL of the website the bug happens on: [https://www.theguardian.com/uk](https://www.theguardian.com/uk)

**Comment by 00sapo** [comment]
Hello, for me it works. Can you try with the latest stale firefox and the latest updated addon?

**Comment by rddunphy** [comment]
Okay, so I'm now on Firefox 106 with the latest Tridactyl beta (1.22.1pre6426-4ec0e5eb), and it still won't work. But if I create a new FF profile, I can't reproduce the problem. So, I assumed it must be a conflict with some other extension - but disabling all other extensions doesn't seem to fix it either. Not sure what else would make the site behave differently between user profiles; if I have time later I'll dig into it a bit more, but if I can't figure it out I'll just close this, since it's not really a huge deal.

**Comment by bovine3dom** [comment]
For me it doesn't work in a fresh profile ... unless I install uBlock. Then it works fine. Which is weird.

**Comment by rddunphy** [comment]
Interesting - I have uBlock Origin installed, too. Maybe it's a specific filter list that makes the difference?

**Comment by bovine3dom** [comment]
That would be surprising - I installed it straight from the add-ons store in a fresh profile, unless you have disabled some of the default filters.

**Comment by rddunphy** [comment]
Nope, I didn't disable any lists. In that case, I have no idea, sorry!
